examples/cmake/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+string(REPLACE " " "" PRODUCT_NAME_NOSPACE ${PRODUCT_NAME})
+
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/FindGeodeNative.cmake.in ${CMAKE_CURRENT_SOURCE_DIR}/Find${PRODUCT_NAME_NOSPACE}.cmake @ONLY)
+
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/../cpp/customserializable/CMakeLists.txt.in ${CMAKE_CURRENT_SOURCE_DIR}/../cpp/customserializable/CMakeLists.txt @ONLY)
```

Comment:
These should not be in this CMake file. They should also be generating into the binary directory. You NEVER modify files in the source directory.